CovertInfiltration/Src/CovertInfiltration/Classes/X2EventListener_Infiltration.uc

```diff
@@ -701,7 +702,7 @@ static protected function EventListenerReturn AfterActionModifyRecoveredLoot (Ob
 	}
 	else 
 	{
-		`Redscreen("No interceptable items for the complication - rescue mission will spawn empty!!!");
+		`CI_Trace("No interceptable loot for the complication - rescue mission will spawn empty unless there is interceptable rewards!");
```

Comment:
`AfterActionModifyRecoveredLoot` will be called after `PreCompleteStrategyFromTacticalTransfer`, so you can check if something else was intercepted already